Sileo/Backend/Download Manager/DownloadManager.swift

```diff
@@ -73,13 +73,7 @@ class DownloadManager: NSObject {
     
     var repoDownloadOverrideProviders: [String: Set<NSObject>] = [:]
     
-    var viewController: DownloadsTableViewController
-    
-    override init() {
-        viewController = DownloadsTableViewController(nibName: "DownloadsTableViewController", bundle: nil)
-        
-        super.init()
-    }
+    var viewController = DownloadsTableViewController(nibName: "DownloadsTableViewController", bundle: nil)
```

Comment:
Can this be `let`?

Reply:
Yep, I wonder why the compiler didn't warn about it